set1/Q1.cpp.txt

```diff
@@ -0,0 +1,27 @@
+// Question link : https://practice.geeksforgeeks.org/problems/k-largest-elements/0
+#include <iostream>
+#include<algorithm>
+#include<vector>
+using namespace std;
+vector<int> arr;
+
+int main()
+{
+	int t, n, k;
```

Comment:
Please name the variables properly.

Reply:
The variable names are according to question stated in the comment section

Reply:
It is T, N and C[i] then. 😂